src/org/opendatakit/briefcase/export/DataType.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2018 Nafundi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.opendatakit.briefcase.export;
+
+import java.util.stream.Stream;
+
+/**
+ * This enum encodes JavaRosa data types as described in {@link org.javarosa.core.model.Constants}
+ */
+enum DataType {
```

Comment:
I think we should declare this enum in JavaRosa, even if we don’t convert JR code to use the enum right away. I’ll make such a PR if I don’t hear disagreement.

Reply:
That's great, thanks!

Reply:
Has the a new version of JR including this enum been released yet?

Reply:
Yes! Still in snapshot but you can use it. The version is 2.10.0-SNAPHOT

Reply:
Ugh... possibly newbie question here, sorry...

I'm getting an error after updating the JR version in the build file:
```
   > Could not find org.opendatakit:opendatakit-javarosa:2.10.0-SNAPHOT.
     Searched in the following locations:
         https://repo.maven.apache.org/maven2/org/opendatakit/opendatakit-javarosa/2.10.0-SNAPHOT/opendatakit-javarosa-2.10.0-SNAPHOT.pom
         https://repo.maven.apache.org/maven2/org/opendatakit/opendatakit-javarosa/2.10.0-SNAPHOT/opendatakit-javarosa-2.10.0-SNAPHOT.jar
         https://oss.sonatype.org/content/groups/public/org/opendatakit/opendatakit-javarosa/2.10.0-SNAPHOT/opendatakit-javarosa-2.10.0-SNAPHOT.pom
         https://oss.sonatype.org/content/groups/public/org/opendatakit/opendatakit-javarosa/2.10.0-SNAPHOT/opendatakit-javarosa-2.10.0-SNAPHOT.jar
```

Reply:
🤔Really? I just tried 
```
   compile(group: 'org.opendatakit', name: 'opendatakit-javarosa', version: '2.10.0-SNAPSHOT') {
        exclude group: 'org.slf4j'
    }
```
and it worked. Maybe some kind of caching? Clean and build again?

You will need to exclude slf4j but that's not what the error was about.

Reply:
Thanks! That did the trick :)

Why exclude slf4j, though?

Reply:
I believe that if the versions in JR and Briefcase get out of sync the build will fail if it's included transitively from JR. They match now so I suppose we can leave it in and make sure to keep the versions in sync. I'd tend to exclude it so we don't have to deal with cryptic errors in the future.